src/mame/namco/namcos21.cpp

```diff
@@ -631,6 +635,7 @@ void namcos21_state::c140_map(address_map &map)
 void namcos21_state::configure_c65_namcos21(machine_config &config)
 {
 	NAMCOC65(config, m_c65, 2048000);
+
 	m_c65->in_pb_callback().set_ioport("MCUB");
 	m_c65->in_pc_callback().set_ioport("MCUC");
 	m_c65->in_ph_callback().set_ioport("MCUH");
```

Comment:
This blank line is definitely unhelpful and against conventions.  We don’t put a blank line between instantiating a device and configuring it.

Reply:
Done.